run-bungeecord.sh

```diff
@@ -44,10 +44,10 @@ case "${TYPE^^}" in
 
   CUSTOM)
     if isURL ${BUNGEE_JAR_URL}; then
-      BUNGEE_JAR=$BUNGEE_HOME/$(basename ${BUNGEE_JAR_URL})
-    elif [[ -f ${BUNGEE_JAR_URL} ]]; then
       echo "Using custom server jar at ${BUNGEE_JAR_URL} ..."
       BUNGEE_JAR=${BUNGEE_JAR_URL}
+    elif [[ -f ${BUNGEE_JAR_URL} ]]; then
+      BUNGEE_JAR=$BUNGEE_HOME/$(basename ${BUNGEE_JAR_URL})
```

Comment:
I'm now thinking either a new variable should be introduced for the file-based jar option, such as `BUNGEE_JAR_PATH` or document that if the user supplies `BUNGEE_JAR` then no further action is needed to obtain the referenced jar. 

Also rather than assume the user placed the jar in `$BUNGEE_HOME` it should just use the path as provided. For example, users of the minecraft images typically mount any number of volumes to supply pre-downloading things like this.

With that, a local variable like `$needsDownload` can be used to convey to the later part of the script if curl needs to be used or if `BUNGEE_JAR` is ready to use as is.